docs/0.24.1/user-guide/integrations.md

```diff
@@ -1,7 +1,6 @@
 ---
 title: Integrations
 layout: doc
-redirect_from: "/docs/integrations/"
```

Comment:
Does anyone know why this is set for so many different versions?

Reply:
I think that's because we want to always show latest integration, and not version specific ones. But to be honest, that's just a guess. It's been a very long time since I've added those.